### PR TITLE
feat: add config to check sentinel result slave node is in master-sla…

### PR DESCRIPTION
**Pull request by mikawudi**
add config to check sentinel result slave node is in master-slave syncing status

Signed-off-by: xujie <mikawudi@qq.com>

### CODE REVIEW

**Comment by mikawudi** [comment]
more info in this issue: https://github.com/redisson/redisson/issues/3345

File: redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import org.redisson.misc.RedissonPromise;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java:37]: don't import classes from spring

File: redisson/src/main/java/org/redisson/config/SentinelServersConfig.java [hunk 2]
```diff
@@ -189,4 +192,13 @@ public SentinelServersConfig setCheckSentinelsList(boolean checkSentinelsList) {
         this.checkSentinelsList = checkSentinelsList;
         return this;
     }
+
+    public boolean isCheckSlaveStatusWithSyncing() {
+        return checkSlaveStatusWithSyncing;
+    }
+
+    public SentinelServersConfig setCheckSlaveStatusWithSyncing(boolean checkSlaveStatusWithSyncing) {
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/config/SentinelServersConfig.java:200]: add comment

**Review by mrniko — CHANGES_REQUESTED** [review]

File: redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java (hunk 1 above)
Comment (mikawudi) [reply redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java:37]: can i use StringUtil from netty?

**Review by mrniko — APPROVED** [review]

**Pull request merged**